Samples/Server/Unity-FractalStream/Assets/Scripts/WebRTCServer.cs

```diff
@@ -147,16 +191,20 @@ private void Update()
         {
             if (!isClosing)
             {
-                transform.position = location;
-                transform.LookAt(lookAt, up);
-
-                // apply stereo projection if needed
-                if (this.IsStereo &&
-                    !stereoLeftProjection.isIdentity &&
-                    !stereoRightProjection.isIdentity)
+                if (!IsStereo)
                 {
-                    this.LeftEye.SetStereoProjectionMatrix(Camera.StereoscopicEye.Left, stereoLeftProjection * this.LeftEye.worldToCameraMatrix);

```

Comment:
why the change here?

Reply:
There are two things:
1. I needed to modify **projectionMatrix** property instead of using **SetStereoProjectionMatrix** method since it doesn't work with rendering to texture.
2. I removed **worldToCameraMatrix** since it inverts the z-axis and results in wrong back face culling. You can see this issue when running our current master.